examples/tools/scene_viewer.rs

```diff
@@ -455,7 +457,9 @@ fn camera_controller(
 
         // Handle mouse input
         let mut mouse_delta = Vec2::ZERO;
-        if mouse_button_input.pressed(options.key_enable_mouse) {
+        if mouse_button_input.pressed(options.mouse_key_enable_mouse)
+            || key_input.pressed(options.keyboard_key_enable_mouse)
```

Comment:
This approach doesn't work too well as you would have to hold down M while using WASD. I think it is better to make the key press a toggle as to whether mouse look is enabled or not. You could use a `Local<bool>` in the camera controller system perhaps.

Reply:
IMO we should just use a resource for this, but the UX argument makes sense.

Reply:
I've implemented that solution now, it seems to work very well!